src/models/HeroUtil.ts

```diff
@@ -15,6 +15,12 @@ class HeroUtil {
     if (hero === "Wrecking Ball") {
       return "wrecking-ball";
     }
+    if (hero === "Junker Queen") {
+      return "junker-queen";
+    }
+    if (hero === "Mccree") {
```

Comment:
Does this need to have the 'c' capitalized to match?
```suggestion
    if (hero === "McCree") {
```